exporters/aerospike/ape.toml.tmpl

```diff
@@ -0,0 +1,156 @@
+
+[Aerospike]
+db_host = "{{.config.aerospike_db_host}}"
+db_port = {{.config.aerospike_db_port}}
+
+# TLS certificates.
+# Supports below formats,
+# 1. Certificate file path                                      - "file:<file-path>"
+# 2. Environment variable containing base64 encoded certificate - "env-b64:<environment-variable-that-contains-base64-encoded-certificate>"
+# 3. Base64 encoded certificate                                 - "b64:<base64-encoded-certificate>"
+# Applicable to 'root_ca', 'cert_file' and 'key_file' configurations.
+
+# root certificate file
+root_ca = ""
+
+# certificate file
+cert_file = ""
+
+# key file
+key_file = ""
+
+# Passphrase for encrypted key_file. Supports below formats,
+# 1. Passphrase directly                                                      - "<passphrase>"
+# 2. Passphrase via file                                                      - "file:<file-that-contains-passphrase>"
+# 3. Passphrase via environment variable                                      - "env:<environment-variable-that-holds-passphrase>"
+# 4. Passphrase via environment variable containing base64 encoded passphrase - "env-b64:<environment-variable-that-contains-base64-encoded-passphrase>"
+# 5. Passphrase in base64 encoded form                                        - "b64:<base64-encoded-passphrase>"
+key_file_passphrase = ""
+
+# node TLS name for authentication
+node_tls_name = ""
+
+# Aerospike cluster security credentials.
+# Supports below formats,
+# 1. Credential directly                                                      - "<credential>"
+# 2. Credential via file                                                      - "file:<file-that-contains-credential>"
+# 3. Credential via environment variable                                      - "env:<environment-variable-that-contains-credential>"
+# 4. Credential via environment variable containing base64 encoded credential - "env-b64:<environment-variable-that-contains-base64-encoded-credential>"
+# 5. Credential in base64 encoded form                                        - "b64:<base64-encoded-credential>"
+# Applicable to 'user' and 'password' configurations.
+
+# database user
+user = ""
+
+# database password
+password = ""
+
+# authentication mode: internal (for server), external (LDAP, etc.)
+auth_mode = ""
+
+# timeout for sending commands to the server node in seconds
+timeout = 5
+
+# Number of histogram buckets to export for latency metrics. Bucket thresholds range from 2^0 to 2^16 (17 buckets).
+# e.g. latency_buckets_count=5 will export first five buckets i.e. <=1ms, <=2ms, <=4ms, <=8ms and <=16ms.
+# Default: 0 (export all threshold buckets).
+latency_buckets_count = 0
+
+# Metrics Allowlist - If specified, only these metrics will be scraped. An empty list will exclude all metrics.
+# Commenting out the below allowlist configs will disable metrics filtering (i.e. all metrics will be scraped).
+
+# Namespace metrics allowlist
+# namespace_metrics_allowlist = []
```

Comment:
Was this a design decision to leave all these allowlist/blocklist out of the configuration?
Note that customers won't be allowed to change all these settings as the ape.toml is automatically generated by the integration.
I know there was some issue templating these arrays into a valid array in ape.toml but it can be easily achieved with something like:

```
namespace_metrics_allowlist = [{{ range $index, $element := .config.namespace_metrics_allowlist}}{{if $index}},{{end}}"{{$element}}"{{end}}]
```
in the `ape.toml.tmpl` file.

There's a useful site that will help you in templating with go template/sprig: https://repeatit.io/#/planet

Reply:
cleaned ape.toml.tmpl by removing all unnecessary fields